test/molecule-role/molecule/default/tests/util.py

```diff
@@ -0,0 +1,13 @@
+import time
+
+def wait_until(somepredicate, timeout, period=0.25, *args, **kwargs):
+  mustend = time.time() + timeout
+  while True:
+    try:
+        somepredicate(*args, **kwargs)
+        return
```

Comment:
you mean `return somepredicate()` ?
or does this do something magic ?

Reply:
I'll rename it. The predicate is not expected to return a value